lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -212,10 +217,9 @@
                                 collection: view.teamsCollection,
                                 breadcrumbs: view.createBreadcrumbs(topic),
                                 title: gettext('Team Search'),
-                                description: interpolate(
-                                    gettext('Showing results for "%(searchString)s"'),
-                                    { searchString: view.teamsCollection.searchString },
-                                    true
+                                description: HtmlUtils.interpolateHtml(
+                                    gettext('Showing results for "{searchString}"'),
+                                    { searchString: view.teamsCollection.searchString }
```

Comment:
Does this contain HTML?  If not, would this be another case of `StringUtils.interpolate()`?